e2e/tests/uniswap_trade_test.rs

```diff
@@ -80,19 +80,19 @@ async fn test_with_ganache() {
     );
     tx!(
         solver,
-        token_a.approve(uniswap_router.address(), to_wei(100))
+        token_a.approve(uniswap_router.address(), to_wei(100_000))
     );
     tx!(
         solver,
-        token_b.approve(uniswap_router.address(), to_wei(100))
+        token_b.approve(uniswap_router.address(), to_wei(100_000))
     );
```

Comment:
Maybe it would be cleaner to have the uniswap liquidity come from a different account that is unrelated to the solver/the settlement. Just so it's clear that with the other PR the solver will make sure the token is approved in uniswap on it's own.